Gemfile.lock

```diff
@@ -668,6 +670,7 @@ DEPENDENCIES
   kaminari (~> 0.17)
   letter_opener (~> 1.4)
   listen (~> 3.1)
+  mail (~> 2.6)
```

Comment:
Not a huge fan of including the `mail` gem just to use it to parse email addresses.. the gem is fairly large and just using it for one method `Mail::Address#address` seems a bit overkill..